src/db_adapters/receipts.rs

```diff
@@ -17,10 +17,98 @@ pub(crate) async fn store_receipts(
     pool: &Pool<ConnectionManager<PgConnection>>,
     receipts: Vec<&near_indexer::near_primitives::views::ReceiptView>,
     block_hash: &str,
+    no_strict_mode: bool,
```

Comment:
Let's not use `no_` for `bool` as it creates double-negative in case of `no_strict_mode = false`